Apps/Admin/Client/Services/ISupportApi.cs

```diff
@@ -13,26 +13,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
-namespace HealthGateway.Admin.Client.Services
-{
-    using System.Collections.Generic;
-    using System.Threading.Tasks;
-    using HealthGateway.Admin.Common.Constants;
-    using HealthGateway.Common.Data.ViewModels;
-    using Refit;
+namespace HealthGateway.Admin.Client.Services;
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using HealthGateway.Admin.Common.Constants;
+using HealthGateway.Common.Data.ViewModels;
+using Refit;
 
+/// <summary>
+/// API to fetch the Messaging Verification from the server.
+/// </summary>
+public interface ISupportApi
+{
     /// <summary>
-    /// API to fetch the Messaging Verification from the server.
+    /// Gets the messaging verification from the server.
     /// </summary>
-    public interface ISupportApi
-    {
-        /// <summary>
-        /// Gets the messaging verification from the server.
-        /// </summary>
-        /// <param name="queryType">queryType.</param>
-        /// <param name="queryString">queryString.</param>
-        /// <returns>The MessagingVerification object.</returns>
-        [Get("/Users?queryType={queryType}&queryString={queryString}")]
-        Task<ApiResponse<RequestResult<IEnumerable<MessagingVerificationModel>>>> GetMedicationVerifications(UserQueryType queryType, string queryString);
-    }
+    /// <param name="queryType">queryType.</param>
+    /// <param name="queryString">queryString.</param>
+    /// <returns>The MessagingVerification object.</returns>
+    [Get("/Users?queryType={queryType}&queryString={queryString}")]
+    Task<ApiResponse<RequestResult<IEnumerable<MessagingVerificationModel>>>> GetMedicationVerifications(UserQueryType queryType, string queryString);
```

Comment:
Similar comment on this, ApiResponse should only be used if we need to do some extra logic based on the Http status code which we shouldn't.

Reply:
@sslaws We need the ApiResponse here because we have the utility that getting the ResultError.
![image](https://user-images.githubusercontent.com/92187402/158483068-21f9c641-0321-46c6-bf87-ea62e3570ed7.png)